index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mobile Version</title>
+</head>
+<body>
+    
+    <div class="toolbar">
+        <div class="colm">My Logo</div>
+        <div class="coli">
```

Comment:
- [ ] Kindly use an `a`  tag for the logo, and it will be your portfolio you can use your name for example instead of `My logo`.